build.gradle.kts

```diff
@@ -0,0 +1,84 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+  kotlin("jvm") version "1.5.10"
+  kotlin("plugin.spring") version "1.5.10"
+  id("maven-publish")
+  id("com.adarshr.test-logger") version "3.0.0"
+  id("com.github.ben-manes.versions") version "0.38.0"
+  id("io.spring.dependency-management") version "1.0.11.RELEASE"
+  id("org.jlleitschuh.gradle.ktlint") version "10.0.0"
+  id("org.owasp.dependencycheck") version "6.1.6"
+  id("org.springframework.boot") version "2.5.0"
+  id("se.patrikerdes.use-latest-versions") version "0.2.16"
+}
+
+group = "uk.gov.justice.hmpps"
+version = "0.0.1-SNAPSHOT"
+
+dependencies {
+  implementation("org.springframework.boot:spring-boot-starter")
+  implementation("com.amazonaws:amazon-sqs-java-messaging-lib:1.0.8")
+
+  implementation("org.springframework.boot:spring-boot-starter-test")
+  testImplementation("org.assertj:assertj-core:3.18.1")
+  testImplementation("org.junit.jupiter:junit-jupiter:5.7.1")
+  testImplementation("org.mockito:mockito-junit-jupiter:3.7.7")
+  testImplementation("com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0")
+}
+
+java.sourceCompatibility = JavaVersion.VERSION_16
+
+tasks.bootJar {
+  enabled = false
+}
+
+repositories {
+  mavenLocal()
+  mavenCentral()
+}
+
+java {
+  withSourcesJar()
+}
+
+publishing {
+  repositories {
+    mavenLocal()
+  }
+  publications {
+    create<MavenPublication>("maven") {
+      groupId = "uk.gov.justice.hmpps"
+      artifactId = "hmpps-spring-boot-sqs"
+      version = "0.0.1-SNAPSHOT"
+
+      from(components["java"])
+    }
+  }
+}
```

Comment:
Temporarily allow publishing locally for testing until we work out how and where to publish permanently.